'jahuty@latest' is not in the npm registry.

When attempting to install this package through npm I get a registry error regarding latest being missing.

```
npm install jahuty --save
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/jahuty - Not found
npm ERR! 404
npm ERR! 404  'jahuty@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-04-21T23_37_28_249Z-debug.log
```

--- a/README.md
+++ b/README.md
@@ -13,9 +13,9 @@ This library requires Node 10+. It may work with earlier versions but has not be
 Install the package with:
 
 ```
-npm install jahuty --save
+npm install @jahuty/jahuty --save
 # or
-yarn add jahuty
+yarn add @jahuty/jahuty
 ```
 
 ## Syntax

